src/azure-cli/azure/cli/command_modules/aro/custom.py

```diff
@@ -66,18 +65,14 @@ def aro_create(cmd,  # pylint: disable=too-many-locals
     if not client_sp:
         client_sp = aad.create_service_principal(client_id)
 
-    rp_client_id = FP_CLIENT_ID
-
+    rp_client_id = os.environ.get('AZURE_FP_CLIENT_ID', 'f1dd0a37-89c6-4e07-bcd1-ffd3d43d8875')
```

Comment:
we should not read AZURE_FP_CLIENT_ID in this codebase, hard code to the uuid.  Probably keeping it as a constant, i.e. `FP_CLIENT_ID = 'f1dd0a37-89c6-4e07-bcd1-ffd3d43d8875'`, is clearer